src/main/java/meditracker/exception/MediTrackerException.java

```diff
@@ -14,4 +14,8 @@ public class MediTrackerException extends Exception {
     public MediTrackerException(String errorContext) {
         super(errorContext);
     }
+
+    public MediTrackerException() {
```

Comment:
Is this empty constructor being used by the code?

Reply:
Resolved in https://github.com/AY2324S2-CS2113T-T09-1/tp/pull/218/commits/f600cce87c24fdfec92f3bd1c50836666b14f4fd